src/rdf/queries.ts

```diff
@@ -182,6 +188,10 @@ const getRegionDimensionsAndFilter = ({
     : undefined;
 };
 
+const cache = new LRUCache<string, Observation[]>({
```

Comment:
I think in the serverless function, this cache object is recreated each instance and thus lost between requests.  I think i've seen articles where they use something like Redis and Upstash to store the cache even when the instance is destroyed

Reply:
As discussed together, we are not in a serverless context when we deploy.